R/isovist.R

```diff
@@ -27,21 +27,23 @@ get_viewpoints <- function(x, density = 1/50) {
 #' @param raylen length of rays
 #'
 #' @return object of class sfc_POLYGON
-#' @import sf
+#' @importFrom rlang !! sym
 #' @export
 get_isovist <- function(occluders, vpoint, rayno = 41, raylen = 100) {
-  if (is.null(occluders)) stop("The occluders object is NULL.")
-  if (is.null(vpoint)) stop("The viewpoint object is NULL.")
+  # Drop any attribute
+  occluders <- sf::st_geometry(occluders)
+  vpoint <- sf::st_geometry(vpoint)
```

Comment:
Not sure I get why there was the check on these arguments being NULL - I think one can assume these are actually sf, sfc or sfg objects as described in the docstring and transform them to sfc via st_geometry?

Reply:
Indeed. `sf::st_geometry()` will complain if NULL geometry is provided. 

Now, while `vpoint` needs to have at least one feature, we might return the maxisovist with a warning if a NULL occluders geometry is provided. What do you think?